internal/operands/common-templates/reconcile_test.go

```diff
@@ -93,12 +92,23 @@ var _ = Describe("Common-Templates operand", func() {
 			Logger:          log,
 			VersionCache:    common.VersionCache{},
 		}
+
+		filteredTemplates = make([]templatev1.Template, 0, len(testTemplates))
+		for _, template := range testTemplates {
+			templateArch, err := GetTemplateArch(&template)
+			Expect(err).ToNot(HaveOccurred())
+
+			if templateArch == architecture.AMD64 {
```

Comment:
Why filter only for AMD64?

Reply:
Maybe group existing tests a little better to make it obvious they are for the single arch template use case? And better separate test vars/helpers for both cases?